src/nxdoc/nuxeo-server/platform-services/conversion.md

```diff
@@ -355,7 +355,7 @@ The Conversion Service supports a global configuration via XML file in order to
       <cachingDirectory>/var/ConversionCache</cachingDirectory>
       <!-- GC interval in minutes (default = 10 minutes ) -->
       <gcInterval>10</gcInterval>
-      <!-- maximum size for disk cache in KB (default to 10*1024) -->
+      <!-- maximum size for disk cache in KB (default to 10240) -->
       <diskCacheSize>1024</diskCacheSize>
```

Comment:
We leave it at 1024 here? or 10240?

Reply:
From what I understand in the code, there is not value set properly in an xml contribution, so I think initially that piece of xml was ment to be simple an example. We can leave it like that. Maybe to avoid confusion we can do 20480 in the second one, to remove all doubts. WDYT?

Reply:
The default is indeed 10*1024, see [GlobalConfigDescriptor.java](https://github.com/nuxeo/nuxeo/blob/55c22582c4f88309ddc34ce304722c3c22265c21/modules/core/nuxeo-core-convert/src/main/java/org/nuxeo/ecm/core/convert/extension/GlobalConfigDescriptor.java#L42), we write it like that in order to improve the readability, as 1 MB == 1024 KB, such writing allows us to read 10 MB directly.

This is better when we're doing two jumps, from MB to B: 10 * 1024 * 1024.

What about keeping the comment as is and replacing the `diskCacheSize` to match the default?

Reply:
Yes this is better

Reply:
@artpick Can you update the PR with the final decision? Thanks :)